code/api/src/seeders/2-products.js

```diff
@@ -13,7 +13,8 @@ module.exports = {
         gender: params.user.gender.female.id,
         image: '/images/stock/belt-female.jpg',
         createdAt: new Date(),
-        updatedAt: new Date()
+        updatedAt: new Date(),
+        score: [1,0]
```

Comment:
This is good. We'll just have to remember that these are random scores right now